bpf/lib/lb.h

```diff
@@ -161,7 +161,7 @@ struct bpf_elf_map __section_maps LB4_MAGLEV_MAP_INNER = {
 	.max_elem	= (LB_MAGLEV_LUT_SIZE / LB_MAGLEV_INNER_ELEMS) + 1,
 	.inner_idx	= NO_PREPOPULATE,
 	.id		= CILIUM_MAP_MAGLEV4,
-	.flags		= BPF_F_INNER_MAP,
+	.flags		= BPF_F_MMAPABLE | BPF_F_INNER_MAP,
```

Comment:
Won't this break for kernels w/o BPF_F_MMAPABLE support when we try to load the datapath?

Reply:
`If the user upgrades kernels coming from <5.5, then both Maglev maps will need to be deleted and recreated when Cilium restarts. Hence, this commit will have an upgrade impact.`

I'm not sure I follow, if the user upgrades kernels, then he would need to reboot the node. The question is if the user already runs on a kernel with 5.5+ and restarts the agent, then there is a small race. You can overcome that by making it opt-in via agent flag (default off), so that you give the user a choice and explain the memory tradeoffs. And when explicitly selected and underlying kernel doesn't support mmap, then agent bails out.